internal/proto/music_player.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package music_player;
+
+option go_package = "/music_player";
+
+service MusicPlayer {
+  rpc play(Empty) returns (PlaylistResponse);
+  rpc pause(Empty) returns (PlaylistResponse);
+  rpc next(Empty) returns (PlaylistResponse);
+  rpc prev(Empty) returns (PlaylistResponse);
+  rpc addSong(SongRequest) returns (PlaylistResponse);
+}
```

Comment:
Please follow the uber style guide for protobuf definitions https://github.com/uber/prototool/blob/dev/style/README.md